includes/html/graphs/application/zfs_l2_abort_lowmem.inc.php

```diff
@@ -8,8 +8,4 @@
 
 $filename = Rrd::name($device['hostname'], ['app', $name, $app->app_id, '_____group2']);
```

Comment:
The upstream Rrd::exists check only checks $rrd_filename.  So we either need to change this to $rrd_filename (and change generic_stats.inc.php as I did in my PR) or have the upstream check if $filename is defined too.  I'm in favor of using a consistent variable name.